NTupleAnalyzerHH_nanoAOD.py

```diff
@@ -1382,20 +1601,20 @@ def MediumIDMuons(T,_met,variation,isdata):
 
 		Pass = True
 		# A preliminary pT cut.
-		Pass *= (_MuonPt[n] > 8)#fixme offline cuts are 20(10)
+		Pass *= (_MuonPt[n] > 10)#fixme offline cuts are 20(10)
 
 		# Eta requirement
 		Pass *= abs(T.Muon_eta[n])<2.4
 
                 # Require Loose muon flag
-	        Pass *= T.Muon_mediumId[n] > 0
+	        Pass *= T.Muon_mediumId[n] > 0 #? does this need to be loose and do we need valid fraction of hits
 
 		# Isolation condition using combined PF relative isolation - 0.25 for loose (98% efficiency), 0.15 for tight (95% efficiency)
 	        correctedIso = T.Muon_pfRelIso04_all[n]
 
 		# Don't apply isolation for QCD studies
 		if nonisoswitch != True:
-			Pass *= correctedIso < 0.25
+			Pass *= correctedIso/_MuonPt[n] < 0.25
```

Comment:
I think the Muon_pfRelIso04_all is already the relative isolation, and therefore does not need to be divided by _MuonPt